applications/luci-app-shadowsocks-libev/root/usr/share/rpcd/acl.d/luci-app-shadowsocks-libev.json

```diff
@@ -0,0 +1,15 @@
+{
+	"luci-app-shadowsocks-libev": {
+		"description": "Grant service list access to LuCI app shadowsocks-libev",
+		"read": {
+			"ubus": {
+				"service": [ "list" ]
+			}
+		},
+		"write": {
+			"file": {
+				"/etc/shadowsocks-libev/*": [ "write" ]
```

Comment:
Quick note: this is not expected.  The shadowsocks-libev package only needs to access uci config file `/etc/config/shadowsocks-libev`.

Reply:
This is used by `form.FileUpload`.